_codux/ui-kit/buttons/buttons.board.tsx

```diff
@@ -0,0 +1,141 @@
+import { createBoard, Variant } from '@wixc3/react-board';
+import { CartIcon, FacebookIcon, PinterestIcon, WhatsAppIcon } from '~/components/icons';
+import { LabelWithArrow } from '~/components/label-with-arrow/label-with-arrow';
+import { TrashIcon } from '../../../src/components/icons/trash-icon';
+import { LockIcon } from '../../../src/components/icons/lock-icon';
+import { ErrorIcon } from '../../../src/components/icons/error-icon';
+import { MenuIcon } from '../../../src/components/icons/menu-icon';
+import { ChevronRightIcon } from '../../../src/components/icons/chevron-right-icon';
+import { ArrowRightIcon } from '../../../src/components/icons/arrow-right-icon';
+import { Kit } from '../ui-kit-utils/kit';
+
+import styles from './buttons.board.module.scss';
+
+export default createBoard({
+    name: 'Buttons',
+    Board: () => (
+        <Kit category="Core Components" title="Buttons">
```

Comment:
It wraps everthing with the "Buttons" title, but there are not only buttons here.

Reply:
Buttons and Icons

Reply:
@Letushev Dana agreed.